.env.example

```diff
@@ -0,0 +1,4 @@
+rollno=
+password=
+ntfyAddr=
```

Comment:
Try to use capital letters for all env vars for consistency.

Reply:
Okay